jsonnet/rules.libsonnet

```diff
@@ -615,5 +615,38 @@ function(params) {
         },
       ],
     },
+    {
+      name: 'openshift-console.rules',
```

Comment:
I'd rather see the rules live in openshift/console-operator if this is possible.

Reply:
@simonpasquier does this mean that I should just change the label or move the metrics code from openshift/console to openshift/console-operator?

Reply:
I meant to say that the console-operator should deploy a PrometheusRule containing the recording rule. Since the metric is owned/managed by the console team, it's more convenient if you also own the rule.
Does it make sense?

Reply:
Yep, that makes sense to me. I opened https://github.com/openshift/console-operator/pull/755 for this. PTAL if this PR is then fine. Thanks.